application/common/models/Email.php

```diff
@@ -60,12 +60,12 @@ public function behaviors()
     }
 
     /**
-     * Attempt to send email. Returns true on success or throws exception.
+     * Attempt to send email. Returns 1 on success or 0 if not sent.
      * DOES NOT QUEUE ON FAILURE
-     * @throws \Exception if the send failed for any reason
-     * @throws EmailDelayedException if email is delayed by `send_after` or `delay_seconds`
+     * @throws \Exception if sending failed
```

Comment:
These comments seem contradictory: does it return 0 or throw an exception if the email did not send?

Reply:
Ahh, I see. The 0 is if it's not time to send it yet, and the exception is if sending was attempted but failed for some reason. That makes sense now.